maybe_rayon/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "maybe_rayon"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
```

Comment:
I'd say let's add `default = ["parallel"]` to make it opt-out.

Reply:
Oh I see it's enabled by default in the plonky2 crate. Makes sense.

Could you do the same for `starky` and `evm` though? Otherwise I guess by default they'd get a mix of parallel code (in starky/evm) and serial code (in plonky2 methods they depend on)?

Reply:
Yes I can do that.